aten/src/ATen/native/TensorTransformations.cpp

```diff
@@ -142,4 +142,67 @@ Tensor flipud(const Tensor& self) {
   return self.flip({0});
 }
 
+Tensor atleast_1d(const Tensor& self) {
+  switch (self.dim()) {
+    case 0:
+      return self.reshape({1});
+    default:
+      return self;
+  }
+}
+
+std::vector<Tensor> atleast_1d(TensorList tensors) {
+  std::vector<Tensor> result(tensors.size());
+  auto transform_lambda = [](const Tensor& input) -> Tensor {
```

Comment:
Can you pass atleast_1d directly instead of wrapping it in a lambda?

Reply:
Actually that was the first thing I tried but it fails as the compiler surprisingly fails to deduce which overload to use.